howtos/documentation-guidelines.md

```diff
@@ -209,25 +216,27 @@ On a technical level, the Optimize team takes screenshots within their [end-to-e
 
 ## Review process
 
-After the proposed change is finished open a GitHub PR and assign at least one reviewer, it is good to pick a reviewer who is expert in the matter of the change. If unsure about who to pick choose one of the corresponding team representatives, and they will take care of delegating the issue:
+After the proposed change is finished, open a GitHub PR and assign reviewers based on your PR template. Most PRs require a technical writing review and a review from the relevant engineering team.
+
+If unsure about who to pick, choose one of the corresponding team representatives, and they will take care of delegating the issue:
 
 - Console: @ultraschuppi
 - Modeler: @camunda/modeling-dev
 - Zeebe: @npepinpe
-- Operate: @ralfpuchert
+- Operate: TBD
 - Tasklist: @camunda/human-task-orchestration
 - Optimize: @RomanJRW
 - Connectors: @camunda/connectors
 - Self-Managed/Distribution: @camunda/distribution
 - InfEx: @camunda/infex-docs-dri
```

Comment:
@Langleu @leiicamundi is this correct? It doesn't come up for me.

Reply:
Hey @akeller , it does exist and I'm in that "sub-team" but let's go ahead and change it to `@camunda/infrastructure-experience`.

@leiicamundi I don't have any permissions anymore but yes idea would be to remove it. I don't think I was ever pinged on that.

Reply:
I've added everyone from InfraEx to `@camunda/infex-docs-dri` and also removed myself from it.

Reply:
@Langleu @leiicamundi are we good to continue using `@camunda/infex-docs-dri` after @maxdanilov's changes? Or do you still want to adjust this to `@camunda/infrastructure-experience`?